frontend/file/build.gradle.kts

```diff
@@ -41,6 +41,11 @@ sourceSets["main"].java { srcDir(additionalSourceDirs) }
 
 val generateMacheteParser =
   tasks.withType<GenerateParserTask> {
+    javaLauncher.set(
```

Comment:
Pls also add a script into CI config that clones JetBrains/gradle-grammar-kit-plugin at your branch and publishes it locally with some unique version + update the version of the plugin in gradle/libs.versions.toml, so that the entire CI pipeline completes successfully. See `Publish ide-probe snapshots locally` step for the general idea

Reply:
In fact, consider just setting the launcher for all `JavaExec` tasks globally, instead of doing that just for individual tasks